src/views/CashlinkReceive.vue

```diff
@@ -173,6 +191,10 @@ export default class CashlinkReceive extends Vue {
         hubApi.on(HubApi.RequestType.LOGIN, handler, handler);
         hubApi.on(HubApi.RequestType.ONBOARD, handler, handler);
         hubApi.checkRedirectResponse();
+
+        this._onResize = this._onResize.bind(this);
+        window.addEventListener('resize', this._onResize);
+        this._onResize();
```

Comment:
I would massively prefer to do breakpoint-display logic with CSS breakpoints instead of with JS.

Reply:
I also generally prefer css media queries and do that if possible.
I this case an `isMobile` property is used to be able to use conditional display via `v-if` in the template to avoid loading both, the desktop and mobile theme, if only one is needed.
`display: none` would not achieve this behavior as images in `<img>` elements are always loaded.
While with css a `background-image` could be applied only when an image is needed, we're using `<img>`s here to enable fade-in after load.